features/raadpleeg-bewoning-met-periode/bewoning-vaste-volgorde-mogelijke-bewoner.feature

```diff
@@ -0,0 +1,169 @@
+# language: nl
+@api
+Functionaliteit: De sortering van mogelijke bewoners op vaste volgorde
+Een mogelijke bewoner is een persoon die (voor een deel) van de gevraagde periode in de onzekerheidsperiode van de gevraagde adreshouding valt
```

Comment:
wat een mogelijke bewoner is, is in principe al in een andere feature beschreven. Wat is de reden om dit in een sortering feature weer uit te leggen?